aten/src/ATen/core/TensorOptions.h

```diff
@@ -135,6 +136,12 @@ struct CAFFE2_API TensorOptions {
     return *this;
   }
 
+  template <typename T>
+  TensorOptions& dtype() {
+    dtype_ = CTypeToScalarType<T>::to();
```

Comment:
Well, I guess we can fix this when @li-roy gets back around to TypeMeta-ifying this class :)